common/logger/src/libra_logger.rs

```diff
@@ -254,6 +295,7 @@ struct LoggerService {
     receiver: Receiver<LogEntry>,
     address: Option<String>,
     printer: Option<Box<dyn Writer>>,
+    facade: Arc<LibraLogger>,
```

Comment:
Why not just call it `logger`?

Reply:
to be more clear since there's technically two different parts of the logger, the Service and the facade